include/CL/sycl/group.hpp

```diff
@@ -182,6 +182,16 @@ struct group {
     detail::parallel_for_workitem_in_group(*this, f);
   }
 
+  /* Comparison operators for group object.
+   */
+  bool operator==(const group &groupB) const {
+    return (group_id == groupB.group_id &&
+	    ndr == groupB.ndr);
+  }
+
+  bool operator!=(const group &groupB) const {
+    return !(*this == groupB);
+  }
```

Comment:
This could be probably factorized on a mixin, for example by inheriting from `detail::add_difference_operator<group>`, while waiting for C++20

Reply:
I am just realizing that this is actually Boost.Operators `equality_comparable<T>` :-)
So just use it.

Reply:
Oh interesting, I 've don that now in the latest push.